### PR TITLE
DOCSUP-8711: Document the sumCount aggregate function and optimize_fuse_sum_count_avg setting

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #21337

### CODE REVIEW

File: docs/en/sql-reference/aggregate-functions/reference/sumcount.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+sumCount(x)
+```
+
+**Arguments** 
+
+-   `x` — Input value, must be [Integer](../../../sql-reference/data-types/int-uint.md), [Float](../../../sql-reference/data-types/float.md), or [Decimal](../../../sql-reference/data-types/decimal.md).
+
+**Returned value**
+
+-   Tuple `(sum, count)`, where `sum` is the sum of numbers and `count` is a counts the number of rows or not-NULL values.
```
Comment (olgarev) [inline docs/en/sql-reference/aggregate-functions/reference/sumcount.md:21]:
```suggestion
-   Tuple `(sum, count)`, where `sum` is the sum of numbers and `count` is the number of rows with not-NULL values.
```

File: docs/ru/sql-reference/aggregate-functions/reference/sumcount.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+sumCount(x)
+```
+
+**Аргументы** 
+
+-   `x` — Входное значение типа [Integer](../../../sql-reference/data-types/int-uint.md), [Float](../../../sql-reference/data-types/float.md), или [Decimal](../../../sql-reference/data-types/decimal.md).
+
+**Возвращаемое значение**
+
+-   Кортеж из элементов `(sum, count)`, где `sum` — это сумма чисел и `count` — количество строк или не-нулевых значений.
```
Comment (olgarev) [inline docs/ru/sql-reference/aggregate-functions/reference/sumcount.md:21]:
```suggestion
-   Кортеж из элементов `(sum, count)`, где `sum` — это сумма чисел и `count` — количество строк с значениями, отличными от `NULL`.
```

File: docs/ru/sql-reference/aggregate-functions/reference/sumcount.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Тип: [Tuple](../../../sql-reference/data-types/tuple.md).
+
+**Пример**
+
+Запрос:
+
+``` sql
+CREATE TABLE test (x Int8) Engine = Log;
```
Comment (olgarev) [inline docs/ru/sql-reference/aggregate-functions/reference/sumcount.md:30]:
```suggestion
CREATE TABLE test (x Nullable(Int8)) Engine = Log;
```

File: docs/ru/sql-reference/aggregate-functions/reference/sumcount.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+INSERT INTO test SELECT number FROM numbers(1, 20);
+SELECT sumCount(x) from test;
```
Comment (olgarev) [inline docs/ru/sql-reference/aggregate-functions/reference/sumcount.md:32]:
```suggestion
CREATE TABLE test (x Nullable(Int8)) Engine = Log;
INSERT INTO test SELECT number FROM numbers(1, 20);
INSERT INTO test VALUES (NULL);
SELECT sumCount(x) from test;
```

File: docs/ru/sql-reference/aggregate-functions/reference/sumcount.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+Результат:
+
+``` text
+┌─sumCount(a)─┐
+│ (210,20)    │
```
Comment (olgarev) [inline docs/ru/sql-reference/aggregate-functions/reference/sumcount.md:39]:
```suggestion
│ (210,21)    │
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -2890,4 +2890,33 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+-   1 — функции с одинаковым аргументом объединяются.
+
+**Пример**
+
+Запрос:
+
+``` sql
+CREATE TABLE fuse_tbl(a Int8, b Int8) Engine = Log;
+SET optimize_fuse_sum_count_avg = 1;
+EXPLAIN SYNTAX SELECT sum(a), sum(b), count(b) from fuse_tbl FORMAT TSV;
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2909]:
```suggestion
EXPLAIN SYNTAX SELECT sum(a), sum(b), count(b), avg(b) from fuse_tbl FORMAT TSV;
```

File: docs/ru/operations/settings/settings.md [hunk 7]
```diff
@@ -2890,4 +2890,33 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+```
+
+Результат:
+
+``` text
+SELECT
+    sum(a),
+    sumCount(b).1,
+    sumCount(b).2
+FROM fuse_tbl
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2919]:
```suggestion
SELECT
    sum(a),
    sumCount(b).1,
    sumCount(b).2,
    (sumCount(b).1) / (sumCount(b).2)
FROM fuse_tbl
```

File: docs/en/operations/settings/settings.md [hunk 8]
```diff
@@ -3000,4 +2999,33 @@ Result:
...
+-   1 — Functions with identical argument are fused.
+
+**Example**
+
+Query:
+
+``` sql
+CREATE TABLE fuse_tbl(a Int8, b Int8) Engine = Log;
+SET optimize_fuse_sum_count_avg = 1;
+EXPLAIN SYNTAX SELECT sum(a), sum(b), count(b) from fuse_tbl FORMAT TSV;
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3018]:
```suggestion
EXPLAIN SYNTAX SELECT sum(a), sum(b), count(b), avg(b) from fuse_tbl FORMAT TSV;
```

File: docs/en/operations/settings/settings.md [hunk 9]
```diff
@@ -3000,4 +2999,33 @@ Result:
...
+```
+
+Result:
+
+``` text
+SELECT
+    sum(a),
+    sumCount(b).1,
+    sumCount(b).2
+FROM fuse_tbl
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3028]:
```suggestion
SELECT
    sum(a),
    sumCount(b).1,
    sumCount(b).2,
    (sumCount(b).1) / (sumCount(b).2)    
FROM fuse_tbl
```

**Comment by adevyatova** [comment]
@akuzm Please, review this PR!

**Review by Avogar — APPROVED** [review]

**Pull request merged**